builtin/providers/aws/resource_aws_sqs_queue.go

```diff
@@ -177,6 +178,9 @@ func resourceAwsSqsQueueRead(d *schema.ResourceData, meta interface{}) error {
 					}
 					d.Set(iKey, value)
 				} else {
+					if iKey == "policy" {
+						*attrmap[oKey] = normalizeJson(*attrmap[oKey])
+					}
```

Comment:
I'm curious why the changes here, do they directly contribute to the S3 notification resource, or is this something you encountered in development that needed patching? I'm curious if it needs testing